src/main/java/com/rallydev/saml/SAMLUtils.java

```
@@ -189,32 +189,49 @@ public static Credential toCredential(X509Certificate certificate) {
 
     public static byte[] loadResource(String locationUri) {
         try {
-            URI uri = URI.create(locationUri);
-            String scheme = uri.getScheme();
-            if (Objects.equals(scheme, "classpath")) {
-                return readClasspathResource(uri.getPath());
-            } else {
-                return Files.readAllBytes(new File(uri.getPath()).toPath());
+            try {
+                URI uri = URI.create(locationUri);
+                String scheme = uri.getScheme();
+                if (Objects.equals(scheme, "classpath")) {
+                    return readClasspathResource(uri.getPath());
+                } else {
+                    return Files.readAllBytes(new File(uri.getPath()).toPath());
+                }
+            } catch (IOException e) {
+                throw new UncheckedIOException(e);
             }
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
+        } catch (Throwable t) {
+            // When Zuul calls this method and it fails with an exception, there is no indication of this in Zuul.
+            // After spending hours trying to track this down with print statements in Zuul,
+            // I added this code to print exceptions to stdout before throwing them to the caller.
+            t.printStackTrace(System.out);
```

this isn't the right way to do this, if the exception is being suppressed in zuul, fix the problem in zuul, never use `printStackTrace`

---

fixed